### PR TITLE
{tools}[intel/2017a] pyenchant v1.6.8, enchant v1.6.1, hunspell v1.6.1

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2053.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/b28f1ad232227a1d017fbc501a83cdf9 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2482.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/80b92a8fb4b70803d58c2c4628797723 for a full test report.

File: easybuild/easyconfigs/e/enchant/enchant-1.6.1-intel-2017a.eb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+description = """Enchant is a library (and command-line program) that wraps a number of different spelling libraries
+ and programs with a consistent interface."""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+
+source_urls = ['https://github.com/AbiWord/enchant/releases/download/enchant-%s/' % '-'.join(version.split('.'))]
+sources = [SOURCE_TAR_GZ]
+
+builddependencies = [('Autotools', '20150215')]
+dependencies = [('hunspell', '1.6.1')]
```
Comment (verdurin) [inline easybuild/easyconfigs/e/enchant/enchant-1.6.1-intel-2017a.eb]: Personally I prefer to have a blank line between the two different types of dependency...

**Comment by boegel** [comment]
Thanks for the review @verdurin!

**Pull request merged**